src/execution/baseline/Adaptor.hpp

```diff
@@ -106,8 +106,33 @@ struct IRAdaptor {
              });
    }
 
-   [[maybe_unused]] static bool cur_arg_is_byval(uint32_t) noexcept { return false; }
-   [[maybe_unused]] static uint32_t cur_arg_byval_align(uint32_t) noexcept { return 0; }
+   [[maybe_unused]] bool cur_arg_is_byval(uint32_t t) noexcept {
```

Comment:
This looks wrong. On AArch64, nothing is passed byval, this only happens for large structs on x86-64.